truenas_installer/installer.py

```diff
@@ -2,7 +2,8 @@
 
 
 class Installer:
-    def __init__(self, version, dmi):
+    def __init__(self, version, dmi, vendor=None):
```

Comment:
This default value will never be used
```suggestion
    def __init__(self, version, dmi, vendor):
```

Reply:
Good catch